mittab/libs/outround_tab_logic/bracket_generation.py

```diff
@@ -0,0 +1,16 @@
+import math
+
+def branch(seed, level, limit):
```

Comment:
Maybe add a clarifying comment here on the variable meanings -- it's not totally clear to me on first-read